### PR TITLE
Changes to solution level bulk suppression command for baselining

### PR DESCRIPTION
**Pull request by mavasani**
**Note:** This change is for milestone 1.2

1. Don't suppress compiler diagnostics when bulk suppressing all active issues in the solution - only suppress analyzer diagnostics in the global suppressions side-car file.
2. Use live diagnostics instead of build diagnostics for computing bulk suppression fix.
3. Show a preview changes dialog for the changes.

Fixes VSO [159792](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems#id=159792&fullScreen=false&_a=edit): Don't suppress FxCop and compiler diagnostics when executing "Suppress active issues" command from solution explorer

### CODE REVIEW

**Comment by mavasani** [comment]
@heejaechang @srivatsn @michaelcfanning @morrisjoe

**Comment by mavasani** [comment]
Ping any feedback?

File: src/VisualStudio/Core/Def/Implementation/TableDataSource/Suppression/VisualStudioSuppressionFixService.cs [hunk 1]
```diff
@@ -126,48 +114,16 @@ public bool RemoveSuppressions(bool selectedErrorListEntriesOnly, IVsHierarchy p
...
-                        {
-                            var tree = await document.GetSyntaxTreeAsync(cancellationToken).ConfigureAwait(false);
-                            var text = await tree.GetTextAsync(cancellationToken).ConfigureAwait(false);
-                            foreach (var diagnostic in group)
-                            {
-                                builder.Add(diagnostic.WithCalculatedSpan(text));
-                            }
-                        }
-                    }
+                    var diagnostics = await _diagnosticService.GetDiagnosticsAsync(solution, project.Id, includeSuppressedDiagnostics: true, cancellationToken: cancellationToken).ConfigureAwait(false);
```
Comment (heejaechang) [inline src/VisualStudio/Core/Def/Implementation/TableDataSource/Suppression/VisualStudioSuppressionFixService.cs]: why include already suppressed diagnsotics?

**Comment by heejaechang** [comment]
:+1: awesome!

File: src/VisualStudio/Core/Def/Implementation/TableDataSource/Suppression/VisualStudioSuppressionFixService.cs (hunk 1 above)
Comment (mavasani) [inline src/VisualStudio/Core/Def/Implementation/TableDataSource/Suppression/VisualStudioSuppressionFixService.cs]: The flag just decide the final filtering applied to diagnostics [here](http://source.roslyn.io/Microsoft.CodeAnalysis.Features/R/f778d33c69a9fe48.html) - the computation of diagnostics always need to include suppressed diagnostics as the result is cached by diagnostic service and any client can request for suppressed diagnostics (e.g. error list when user wants to see suppressed issues).

However, you are right that this code path doesn't need suppressed diagnostics. I will clean it up in next change, thanks.

**Comment by mavasani** [comment]
We have decided to postpone this design change until there is customer feedback.

**Pull request closed without merging**